modules/monitoring-resizing-a-persistent-storage-volume.adoc

```diff
@@ -0,0 +1,203 @@
++
+[source,terminal]
+----
+$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
+----
+
+.. Add a new storage size for the PVC configuration for the component under `data/config.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    <component>: <1>
+      volumeClaimTemplate:
+        spec:
+          storageClassName: <storage_class> <2>
+          resources:
+            requests:
+              storage: <amount_of_storage> <3>
+----
+<1> Specify the core monitoring component.
+<2> Specify the storage class.
+<3> Specify the new size for the storage volume.
++
+The following example configures a PVC that changes the local persistent storage to 100 gigabytes for the Prometheus instance that monitors core {product-title} components:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      volumeClaimTemplate:
+        spec:
+          storageClassName: local-storage
+          resources:
+            requests:
+              storage: 100Gi
+----
++
+The following example changes a PVC that changes local persistent storage for Alertmanager to 40 gigabytes:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    alertmanagerMain:
+      volumeClaimTemplate:
+        spec:
+          storageClassName: *local-storage*
+          resources:
+            requests:
+              storage: 40Gi
+----
+
+** *To configure a PVC for a component that monitors user-defined projects*:
++
+[NOTE]
+====
+You can resize the volumes for the Alertmanager, Thanos Ruler and Prometheus instances that monitor user-defined projects.
+====
++
+.. Edit the `user-workload-monitoring-config` `ConfigMap` object in the `openshift-user-workload-monitoring` project:
++
+[source,terminal]
+----
+$ oc -n openshift-user-workload-monitoring edit configmap user-workload-monitoring-config
+----
+
+.. Update the PVC configuration for the monitoring component under `data/config.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: user-workload-monitoring-config
+  namespace: openshift-user-workload-monitoring
+data:
+  config.yaml: |
+    <component>: <1>
+      volumeClaimTemplate:
+        spec:
+          storageClassName: <storage_class> <2>
+          resources:
+            requests:
+              storage: <amount_of_storage> <3>
+----
+<1> Specify the core monitoring component.
+<2> Specify the storage class.
+<3> Specify the new size for the storage volume.
++
+The following example changes the PVC size to 100 gigabytes the Prometheus instance that monitors user-defined projects:
```

Comment:
The following example changes the PVC size to 100 gigabytes for the Prometheus instance that monitors user-defined projects

Reply:
fixed